src/path_build.h

```diff
@@ -10,6 +10,13 @@
 #ifndef PATH_BUILD_H
 #define PATH_BUILD_H
 
+static const Money PATH_CONSTRUCT_COST_FLAT         (1200);  ///< How much it costs to build a single section of flat path.
+static const Money PATH_CONSTRUCT_COST_RAMP         (1400);  ///< How much it costs to build a single section of ramped path.
+static const Money PATH_CONSTRUCT_COST_ELEVATED_FLAT(3200);  ///< How much it costs to build a single section of elevated flat path.
+static const Money PATH_CONSTRUCT_COST_ELEVATED_RAMP(3800);  ///< How much it costs to build a single section of elevated ramped path.
+static const Money PATH_CONSTRUCT_COST_CHANGE       ( 400);  ///< How much it costs to change the type of a single existing path segment.
+static const Money PATH_CONSTRUCT_COST_RETURN       (-800);  ///< How much money deletion of a single path segment returns.
```

Comment:
```suggestion
static const Money PATH_CONSTRUCT_COST_RETURN       (-800);  ///< How much it costs to delete a single path segment.
```
The comment describes the unit of `PATH_CONSTRUCT_COST_RETURN` value. Live gets much simpler if you have the same unit everywhere and you seem to be doing that. The fact that its value is a negative cost so you actually return money doesn't change the unit of the value.

Compare:
`The forward speed of the car is currently -10` means it's currently driving backwards while `The backwards speed of the car is currently -10` means that the car is actually driving forward.

You start with the direction indicated in the textual description of the unit of the speed, and then change direction for the actual movement based on the sign of the value.